cranelift-codegen/meta/src/shared/settings.rs

```diff
@@ -21,8 +21,9 @@ pub(crate) fn define() -> SettingGroup {
         r#"
         Run the Cranelift IR verifier at strategic times during compilation.
 
-        This makes compilation slower but catches many bugs. The verifier is
-        disabled by default, except when reading Cranelift IR from a text file.
+        This makes compilation slower but catches many bugs. The verifier is always enabled by
+        default, which is useful during development. It is recommended to manually disable it in
```

Comment:
I think `It is recommended to manually disable it in production environments.` is even a bit strong. I'd be lightly surprised if folks disabled verification in AOT contexts, for example. In fact, I can't think of guidance to offer other than the general "turn off optional features when you're tuning Cranelift for codegen speed". What do you think about just removing the last sentence here?

Reply:
Good point!